src/async-components/views/dialogs/secretstorage/CreateSecretStorageDialog.js

```diff
@@ -259,22 +284,15 @@ export default class CreateSecretStorageDialog extends React.PureComponent {
                 await cli.bootstrapSecretStorage({
                     authUploadDeviceSigningKeys: this._doBootstrapUIAuth,
                     createSecretStorageKey: async () => this._recoveryKey,
-                    setupNewKeyBackup: this.state.useKeyBackup,
+                    setupNewKeyBackup: true,
```

Comment:
do we not support this option anymore?

Reply:
This will shortly be part of dialog for setting up SSSS rather than cross-signing, so you'll either set up both SSSS storage and key backup or neither, as they're both basically server-side key backup. They'll be independent of cross-signing.